docs/map-making/upgrading-1-8-maps.md

```diff
@@ -1,14 +1,22 @@
 A list of (case sensitive) changes, to upgrade a map from a TripleA version to another one.
 
-When changes are required, all occurrences of the code before the colon must be substituted with the code after the colon (if there is no code after the colon, the code must be substituted with nothing (deleted)). In any code, the "\*" symbol means whatever (the change or deletion is to be applied to every given code having whatever character or characters in place of the "\*").
+When changes are required, all occurrences of the code before the `>>` must be substituted
```

Comment:
I preferred ":", instead of ">>". I'm not sure if ">>" is meaningful for reasons I'm not aware. It just makes me thing to "skip forward". Alternatively to the ":", I might have chosen the "|", as symbol. Just stating my personal preference.

Reply:
The `:` was really hard to see. The double ">>" is an arrow. A single ">" looks like an inequality sign, and an actual arrow "->" rendered kinda weird.